main.py

```diff
@@ -182,18 +198,44 @@ def main_reconcile(path: Path, user_inputs: types_.UserInputs) -> dict:
     return run_reconcile(clients=clients, user_inputs=user_inputs)
 
 
+@execute_in_tmpdir
+def main_checks(path: Path, user_inputs: types_.UserInputs) -> bool:
+    """Checks to make sure that the repository is in a consistent state.
+
```

Comment:
I'm wondering if this explanation might help :)
```suggestion
    """Checks to make sure that the repository is in a consistent state.
    
      The repository is in a consistent state if there is a `upload-charm-docs/base-content` tag exists in the `base_branch` and the commit belongs to the `base_branch`. If no tag exists, the `upload-charm-docs/base-content` tag will be created for the current commit.
```